keras/src/ops/numpy.py

```diff
@@ -2801,8 +2801,8 @@ def imag(x):
 
 
 class Isclose(Operation):
-    def call(self, x1, x2):
-        return backend.numpy.isclose(x1, x2)
+    def call(self, x1, x2, rtol=1e-5, atol=1e-8, equal_nan=False):
```

Comment:
If you change the signature in `keras.ops`, you need to change the backend function for all 4 backends, not just numpy.

Reply:
Of course, silly mistake. I am still finding my way into the library. Fixed now!